app.js

```diff
@@ -188,6 +186,15 @@ app.post('/migrate', function (req, res) {
 
                         return jiraFetchIssues(project, firstIssue, lastIssue)
                             .then(function () {
+                                project.projects = Array.from(project.projects);
```

Comment:
json dumping was not supporting set Data Structure. 
I did try it and nothing broke as such. So, I'm not sure about how it'll break.